web_src/js/components/ActivityHeatmap.vue

```diff
@@ -36,10 +40,6 @@ export default {
       'var(--color-primary-dark-4)',
     ],
     endDate: new Date(),
-    locale: {
-      contributions: 'contributions',
-      no_contributions: 'No contributions',
-    },
```

Comment:
I haven't tested , not sure whether these removals are by purpose.

Otherwise I have no other questions.

Reply:
just moved to another file